docs/reference/run.md

```diff
@@ -900,6 +901,11 @@ If you do block IO in the two containers at the same time, by, for example:
 You'll find that the proportion of time is the same as the proportion of blkio
 weights of the two containers.
 
+The `--blkio-weight-device` flag can set the weight of a particular device to a value between 10 to 1000. With this, override the default value of group weight as specified by blkio.weight.
+For example, the command below create one container and set weight of /dev/sda 
```

Comment:
I think we could use *two* examples, one just with `--blkio-weight-device` and one with both flags. Something like;

```Markdown
The `--blkio-weight-device` flag allows you to set the weighting for a specific device.
The value is a string, containing the name of the device and the desired weighting,
using a colon as separator.

For example, to set the weight of the `/dev/sda` device to 200, use this command:

    $ docker run -it \
        --blkio-weight-device "/dev/sda:200" \
        ubuntu

If both `--blkio-weight` and `--blkio-weight-device` are specified, `--blkio-weight`
is used as the default weight and `--blkio-weight-device` overrides the weighting
for a specific device.

The following example uses a default weighting of 300, and a weighting of 200
for `/dev/sda`:

docker run -it \
    --blkio-weight 300 \
    --blkio-weight-device "/dev/sda:200" \
    ubuntu
```